src/main/java/net/snowflake/client/jdbc/SnowflakeType.java

```diff
@@ -190,6 +192,7 @@ public static int convertStringToType(String typeName) {
         retval = Types.JAVA_OBJECT;
         break;
       case "array":
+      case "vector":
```

Comment:
Shouldn't we use EXTRA_TYPES_VECTOR here? I'm not convinced.

Reply:
you are right, changed

Reply:
Was there a mistake in tests or it was not covered?

Reply:
it was not covered. I think it is currently not used. It would be used when we would enable bindings for vector type.